Nodejs/Product/Nodejs/Options/NodejsNpmOptionsControl.cs

```diff
@@ -33,12 +33,14 @@ public partial class NodejsNpmOptionsControl : UserControl {
 
         internal void SyncControlWithPageSettings(NodejsNpmOptionsPage page) {
             _showOutputWhenRunningNpm.Checked = page.ShowOutputWindowWhenExecutingNpm;
+            _enableAutoTypeAcquisition.Checked = page.EnableAutomaticTypeAcquisition;
```

Comment:
I see `AutoType`, `AutoTyping`, and `AutoTypings` being used  - let's stick with the latter-most form to keep things consistent.